theme/src/resources/templates/new_request_page.hbs

```diff
@@ -278,5 +280,52 @@
 				);
 			}
 		}
+
+		// Custom article suggestion
+
+		const subjectField = document.getElementById('request_subject');
+
+		const searchProductLabel = {
+			prd_liferay_analytics_cloud: 'Analytics Cloud',
+			prd_liferay_commerce: 'Commerce 1.0',
+			prd_liferay_dxp_7_0: 'DXP 7.0',
+			prd_liferay_dxp_7_1: 'DXP 7.1',
+			prd_liferay_dxp_cloud: 'DXP Cloud',
+			prd_liferay_portal_6_1: 'Portal 6.1',
+			prd_liferay_portal_6_2: 'Portal 6.2'
+		};
+
+		// Append an element to render the article suggestions
+
+		$('.request_subject').append('<div id="articleSuggestionWrapper"></div>');
+
+		const setArticleSuggestionInnerHTML = function(html) {
+			const articleSuggestionWrapper = document.getElementById('articleSuggestionWrapper');
+
+			if (articleSuggestionWrapper) {
+				articleSuggestionWrapper.innerHTML = html;
+			}
+		}
+
+		const onRequestSubjectKeyup = function() {
+			const productLabel = searchProductLabel[productInputField.value] || '';
+
+			if (subjectField.value) {
+				Liferay.displayArticleSuggestion(
+					subjectField.value,
+					productLabel,
+					'{{help_center.locale}}',
+					'{{dc 'ticket_view-suggested_articles'}}', setArticleSuggestionInnerHTML
+				);
+			} else {
+				setArticleSuggestionInnerHTML('');
```

Comment:
I intentionally set the innerHtml to `''` in the else statement, because I find that if I preset it before the `if` clause, there's this flash when the content is rendered. Any improvement suggestions welcome!